server/views/partials/timelineEvents/booking_changed.njk

```diff
@@ -0,0 +1,12 @@
+<p class="govuk-body">The placement at {{ premises.name }} has been changed:</p>
+<ul class="govuk-list govuk-list--bullet">
+    {% if previousExpectedArrival %}
+        <li>Arrival date changed from {{ previousExpectedArrival }} to {{ expectedArrival }}</li>
+    {% endif %}
+    {% if previousExpectedDeparture %}
+        <li>Departure date changed from {{ previousExpectedDeparture }} to {{ expectedDeparture }}</li>
+    {% endif %}
+    {% if previousCharacteristics %}
+        <li>Room criteria changed from {{ previousCharacteristics }} to {{ characteristics }}</li>
+    {% endif %}
```

Comment:
It might be a strange edge-case, but ATM, we don't prevent an update being made where there are no changes at all!  In this case, the timeline event displays nothing. 
Perhaps it would be nice to add a 'No change' text if not  `previousExpectedArrival or previousExpectedDeparture or previousCharacteristics`